python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -10,6 +10,28 @@
 
 # ==============================================================================
 
+class Conv1dNoPaddingModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.weights = torch.ones(768, 768, 1)
+        self.bias = torch.ones(768)
+
+    @export
+    @annotate_args([
+        None,
+        ([1, 768, 768], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.convolution(x, self.weights, None, [1], [0], [1], False, [0], 1)
```

Comment:
Can we have a test with argument `transposed` being equal to True to make sure that this works ?
(Alternatively, exclude that case in the pattern)

Reply:
Sure! Will do!